src/constants.js

```diff
@@ -20,10 +20,8 @@ import {
   _SENTRY_DSN as SENTRY_DSN,
 } from './config';
 
-hathorLib.storage.setStore(new AsyncStorageStore());
-hathorLib.storage.setItem('wallet:server', 'https://mobile.wallet.hathor.network/v1a/');
-hathorLib.network.setNetwork('mainnet');
-
+export const STORE = new AsyncStorageStore()
+hathorLib.storage.setStore(STORE);
```

Comment:
We don't need to `setNetwork(...)` anymore? Is it being set somewhere else?

Reply:
The wallet is supposed to call `setNetwork()`.

Reply:
Yeah, the `Connection` object sets the network when it's started.